### PR TITLE
Policy location update

### PR DESCRIPTION
**Pull request by joinimran**
Updated a location for policy for AD Server.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5673

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The following table lists the Group Policy settings that you can configure for W
 
 > [!NOTE]
 > Starting with Windows 10, version 1709, the location of the PIN complexity section of the Group Policy is: **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **PIN Complexity**.
+> In AD Server Group enable **Turn on Convenience pin sign-in** under **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **Logon**.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]: I might be a little bit more concerned than strictly needed about readability/natural language flow. Anyway:
```suggestion
> In the AD Server Group, enable **Turn on Convenience pin sign-in** found underneath **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **Logon**.
```

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Comment by joinimran** [comment]
@mapalko I have applied changes in the content based on your recommendations.

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 2]
```diff
@@ -30,14 +30,16 @@ You can create a Group Policy or mobile device management (MDM) policy that will
...
+>
 >Use **PIN Complexity** policy settings to manage PINs for Windows Hello for Business.
+>
+> Starting with Windows 10, version 1709, the location of the PIN complexity section of the Group Policy is: **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **PIN Complexity**.
+
  
 ## Group Policy settings for Windows Hello for Business
 
 The following table lists the Group Policy settings that you can configure for Windows Hello use in your workplace. These policy settings are available in both **User configuration** and **Computer Configuration** under **Policies** &gt; **Administrative Templates** &gt; **Windows Components** &gt; **Windows Hello for Business**.
-
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]: The PIN complexity note is more relevant for this section and shouldn't be moved.

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 3]
```diff
@@ -30,14 +30,16 @@ You can create a Group Policy or mobile device management (MDM) policy that will
 >
 >Beginning in version 1607, Windows Hello as a convenience PIN is disabled by default on all domain-joined computers. To enable a convenience PIN for Windows 10, version 1607, enable the Group Policy setting **Turn on convenience PIN sign-in**. 
 >
+> In the AD Server Group Policy, enable **Turn on Convenience pin sign-in** found underneath **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **Logon**.
+>
 >Use **PIN Complexity** policy settings to manage PINs for Windows Hello for Business.
+>
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]: See my comment below. I don't think should be moved

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 4]
```diff
@@ -30,14 +30,16 @@ You can create a Group Policy or mobile device management (MDM) policy that will
 >
 >Beginning in version 1607, Windows Hello as a convenience PIN is disabled by default on all domain-joined computers. To enable a convenience PIN for Windows 10, version 1607, enable the Group Policy setting **Turn on convenience PIN sign-in**.
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]:
```suggestion
>Beginning in version 1607, Windows Hello as a convenience PIN is disabled by default on all domain-joined computers. To enable a convenience PIN for Windows 10, version 1607, enable the Group Policy setting **Turn on convenience PIN sign-in** found underneath **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **Logon**.
```

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md [hunk 5]
```diff
@@ -30,14 +30,16 @@ You can create a Group Policy or mobile device management (MDM) policy that will
 >
 >Beginning in version 1607, Windows Hello as a convenience PIN is disabled by default on all domain-joined computers. To enable a convenience PIN for Windows 10, version 1607, enable the Group Policy setting **Turn on convenience PIN sign-in**. 
 >
+> In the AD Server Group Policy, enable **Turn on Convenience pin sign-in** found underneath **Computer Configuration** &gt; **Administrative Templates** &gt; **System** &gt; **Logon**.
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]: I would merge this with the previous statement as shown in my suggestion.

**Comment by joinimran** [comment]
@mapalko I have committed your suggestion.

**Comment by e0i** [comment]
@mapalko Do you approve the revised changes?

File: windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md (hunk 5 above)
Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-manage-in-organization.md]: I could be mistaken, but shouldn't this conversation thread be marked [Outdated] if the line above had been moved (merged) into the previous statement, as suggested?

**Comment by mapalko** [comment]
Is the AD server policy statement needed?

**Comment by illfated** [comment]
Well, according to @blouwagie, the original poster of issue ticket #5673, the current document is not as helpful for domain server administrators as it is for client computer administrators.

**Comment by joinimran** [comment]
@mapalko - Any further input required from my side?

**Comment by e0i** [comment]
@Dansimp The PR that has been approved by the document author is ready to be merged. Thanks.

**Comment by e0i** [comment]
Closing as the PR has reportedly been depreceated with Matthew Palko's further updates to the article (_via Imran_).

**Pull request closed without merging**